charts/amoro/values.yaml

```diff
@@ -171,6 +171,13 @@ optimizer:
     name: ~
     # properties for optimizer container.
     properties: ~
+  kubernetes:
+    # enable the kubernetes optimizer container
+    enabled: true
+    properties: {
+      kube-config-path: "～/.kube/config",
+      image: "arctic163/amoro:0.6"
```

Comment:
Don't use hard code version in the default values.yaml

Reply:
I think this is just a means of filling the field by default, as long as the user writes the value will be overwritten as the true value, if not, follow the default